covsirphy/phase/trend.py

```diff
@@ -69,8 +69,8 @@ def run(self):
         dataframe_dict = {func: self._run(func=func) for func in (L, N)}
         score_dict = {
             func: self._rmsle(fit_df) for (func, fit_df) in dataframe_dict.items()}
-        # Seclect the best dataframe
-        if 0 < score_dict[L] < score_dict[N]:
+        # Select the best dataframe
+        if ((score_dict[L] < score_dict[N]) and (score_dict[L] > 0)) or (score_dict[N] == 0):
```

Comment:
It works, but we can use `if 0 < score_dict[L] < score_dict[N] or not score_dict[N]:` to simplify this line.

Reply:
Yes of course, I wrote it C-like 🙂